internal/config/config.go

```diff
@@ -51,6 +51,7 @@ type peer struct {
 	NodeSelectors []nodeSelector `yaml:"node-selectors"`
 	Password      string         `yaml:"password"`
 	BFDProfile    string         `yaml:"bfd-profile"`
+	MultiHop      *bool          `yaml:"multi-hop"`
```

Comment:
Thinking of a future API extension where users may be able to set the maximum allowed number of hops (integer), would it be best to rename this parameter to "EnableMultiHop"?